portfolio/src/main/webapp/script.js

```diff
@@ -93,6 +93,22 @@ function setRandomImage() {
   galleryItemsIndex ++;
 }
 
+/** Function to get a random index if the comments needs to be displayed */
+// function getRandomIndex(Object) {
+//   // Get's a random index from gallery items
+//   return Math.floor(Math.random() * Object.length);
+// }
+
+function setWebpageDefaults() {
+  setRandomImage();
+  fetch('/data').then(response => response.json()).then((data) => {
```

Comment:
```suggestion
  fetch('/comments').then(response => response.json()).then((data) => {
```
It might be nice to rename this servlet hook to "comments". Data is pretty general and we should try to be as specific as we can.

Reply:
To add to this, it's not unusual for APIs to have a structure that is guided by the possible actions. So in the case of comments you might follow a general CRUD model (create, read, update and delete) as four possible actions a user could take. Those might be organized like:

/comments/add
/comments/list
/comments/update
/comments/delete

So to add a comment I might call:

https://my.api.com/v1/whatever/comments/add?title=foo&message=bar

but to delete a comment I might do:

https://my.api.com/v1/whatever/comments/delete?id=123


Not sure what the end game here is, but thought I'd mention it.

Reply:
Of course, the /data name was default on the Java file, but it surely can be changed!